index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Assignment 1 - INFO2340 </title>
+    <meta charset="utf-8">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+	<script src="js/jquery-3.4.1.js"></script>
+	<script src="js/assignment1.js"></script>
+</head>
+
+<body>
+<!-- Scott Fincham
+     assignment1
+     INFO2340
+     Thoendel
+    Winter 2019
+    -->
+<h1> Greetings</h1>
+<p>Hello, my name is <span id="first-name"></span><span id="last-name">!</span></p>
+<p>I am a student at <span class="mcc">Metropolitan Community College</span> in Omaha, Nebraska!</p>
+
+<p>My hobbies include:<br>
+    <ul>
+        <li id="hobby">Videography</li>
+        <li id="hobby">Photography</li>
+        <li id="hobby">Voice Over Acting</li>
```

Comment:
These hobbies should be similar to the name spans - not changed from the figure in the assignment. We want those changes to happen with the custom js by calling jQuery functions.